### PR TITLE
Created some tools to help parse raw cookies, figure out if expires > 1 ...

### PR DESCRIPTION
**Pull request by daviddahl**
...year, log them as neverexpire cookies

### CODE REVIEW

**Comment by monicachew** [comment]
Assigning PR to myself for review. Can you please merge to pick up changes from 14d12cff1805a8be0c245c81106d0d6b12b7ed74?

File: lib/main.js [hunk 1]
```diff
@@ -7,6 +8,65 @@ let eTLDService = Cc["@mozilla.org/network/effective-tld-service;1"]
...
+
+function oneYearFromNow()
+{
+  let nextYear = Date.now() + (1000 * 60 * 60 * 24 * 365);
+  return nextYear;
+}
+
+function parseDate(aDateStr)
+{
+  let dateStr;
```
Comment (monicachew) [inline lib/main.js:24]: Simpler to do 

let dateStr = aDateStr.replace(/-/g, " ")

The if check doesn't really save time here.

File: lib/main.js [hunk 2]
```diff
@@ -7,6 +8,65 @@ let eTLDService = Cc["@mozilla.org/network/effective-tld-service;1"]
...
+  try {
+    return Date.parse(dateStr);
+  }
+  catch (ex) {
+    console.error(ex);
+    return "Undetermined Date"
+  }
+}
+
+function parseCookie(aCookieStr)
```
Comment (monicachew) [inline lib/main.js:40]: The parseAndLogCookie function already parses all properties, do we need this one as well?

**Comment by monicachew** [comment]
dumb test @monicachew

File: lib/main.js (hunk 2 above)
Comment (daviddahl) [inline lib/main.js:40]: No. I was writing the same patch not knowing about your earlier patch

**Comment by monicachew** [comment]
Closing since it's in your branch.

**Pull request closed without merging**